platform/platform_linux.cpp

```diff
@@ -40,6 +43,28 @@ bool IsEulaExist(string const & directory)
 {
   return Platform::IsFileExistsByFullPath(my::JoinFoldersToPath(directory, "eula.html"));
 }
+
+string HomeDir()
+{
+  char const * homePath = ::getenv("HOME");
+  return homePath != nullptr ? string(homePath) : "";
+}
+
+// Returns the default path to the writable dir, creating the dir if needed.
+// An exception is thrown if the default dir is not already there and we were unable to create it.
+string DefaultWritableDir()
+{
+  auto const home = HomeDir();
+  string const result = my::JoinFoldersToPath({home, ".local", "share"}, "MapsWithMe");
```

Comment:
Are you sure that in case when HOME variable is not set, you want to use
```
/.local/share/MapsWithMe
```
as a writable dir?

Reply:
What do you suggest instead? I did not change the existing behaviour.

Reply:
I suggest to do nothing in this case, as this is quite strange environment.